### PR TITLE
fix: Add id and fix date on message sent to Kafka queue for datalake

### PR DESCRIPTION
**Pull request by andrea-putzu**

### CODE REVIEW

File: src/main/protobuf/v1/relationship.proto [hunk 1]
```diff
@@ -30,7 +30,6 @@ message PartyRelationshipV1 {
     SUSPENDED = 3;
     DELETED = 4;
     REJECTED = 5;
-    TOBEVALIDATED = 6;
```
Comment (galales) [inline src/main/protobuf/v1/relationship.proto:33]: ℹ️ remember that you can use position `6` in the future only if there is not data containing the `TOBEVALIDATED` value

File: src/test/scala/it/pagopa/interop/partymanagement/ProjectionContractsHandlerSpec.scala [hunk 2]
```diff
@@ -244,7 +244,7 @@ class ProjectionContractsHandlerSpec
         "fileName",
         "contentType",
         UUID.randomUUID(),
-        OffsetDateTime.now
+        OffsetDateTime.now.toInstant.atZone(ZoneId.of("Z")).toOffsetDateTime
```
Comment (galales) [inline src/test/scala/it/pagopa/interop/partymanagement/ProjectionContractsHandlerSpec.scala]:
```suggestion
        OffsetDateTime.now(ZoneOffset.UTC)
```

File: src/main/scala/it/pagopa/interop/partymanagement/service/impl/OffsetDateTimeSupplierImp.scala [hunk 3]
```diff
@@ -2,8 +2,8 @@ package it.pagopa.interop.partymanagement.service.impl
 
 import it.pagopa.interop.partymanagement.service.OffsetDateTimeSupplier
 
-import java.time.OffsetDateTime
+import java.time.{OffsetDateTime, ZoneOffset}
 
 case object OffsetDateTimeSupplierImp extends OffsetDateTimeSupplier {
-  override def get: OffsetDateTime = OffsetDateTime.now()
+  override def get: OffsetDateTime = OffsetDateTime.now().withOffsetSameInstant(ZoneOffset.UTC)
```
Comment (galales) [inline src/main/scala/it/pagopa/interop/partymanagement/service/impl/OffsetDateTimeSupplierImp.scala]:
```suggestion
  override def get: OffsetDateTime = OffsetDateTime.now(ZoneOffset.UTC)
```

**Review by galales — APPROVED** [review]

File: src/main/protobuf/v1/relationship.proto (hunk 1 above)
Comment (andrea-putzu) [reply src/main/protobuf/v1/relationship.proto:33]: Yes, is just a rollback to limit the pr scope

**Pull request merged**